app.py

```diff
@@ -36,14 +37,14 @@ def _main():
     args = parser.parse_args()
     # Allow users to pick storage location we may want to
     # store our models in google cloud storage or azure instead of s3 in the future
-    if args.modelstore=="s3":
+    if args.modelstore=="s3" or os.getenv("LAD_MODEL_STORE")== "s3":
```

Comment:
missing space ` ` after `MODEL_STORE")`

Reply:
I'm removing this in another PR. I'm going to have this setup in config.py

Reply:
Okay I've removed this and put it in config.py